virttest/libvirt_xml/network_xml.py

```diff
@@ -173,13 +173,15 @@ class DNSXML(base.LibvirtXMLBase):
             List of host name
     """
 
-    __slots__ = ('dns_forward', 'txt', 'forwarders', 'srv',
+    __slots__ = ('enable', 'dns_forward', 'txt', 'forwarders', 'srv',
```

Comment:
add one line explanation above for newly added attribute name enable

Reply:
Updated, Thank you!